In [132]:
from google.colab import drive
drive.mount('/content/drive')

Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount("/content/drive", force_remount=True).


# **COLLECTING DATAS AND CONVERTING THEM INTO ML USABLE FORMAT**




*IMPORTING LIBRARIES*

In [133]:
#imports
import os
import pandas as pd
import numpy as np
import random
random.seed(1)

In [134]:
df=pd.read_csv(f'/content/drive/MyDrive/ML DATASETS/BRFSS22.CSV')

In [135]:
#NO. OF ROWS AND COLUMN
print('Rows/Rcords :-',df.shape[0])
print('Columns/features :-',df.shape[1])

Rows/Rcords :- 445132
Columns/features :- 326


In [136]:
#Checking if loaded data is in the correct format
pd.set_option('display.max_columns', 500)
df.head()

_STATE  FMONTH    IDATE  IMONTH  IDAY  IYEAR  DISPCODE       SEQNO  \
0       1       1  2032022       2     3   2022      1100  2022000001   
1       1       1  2042022       2     4   2022      1100  2022000002   
2       1       1  2022022       2     2   2022      1100  2022000003   
3       1       1  2032022       2     3   2022      1100  2022000004   
4       1       1  2022022       2     2   2022      1100  2022000005   

         _PSU  CTELENM1  PVTRESD1  COLGHOUS  STATERE1  CELPHON1  LADULT1  \
0  2022000001       1.0       1.0       NaN       1.0       2.0      1.0   
1  2022000002       1.0       1.0       NaN       1.0       2.0      1.0   
2  2022000003       1.0       1.0       NaN       1.0       2.0      1.0   
3  2022000004       1.0       1.0       NaN       1.0       2.0      1.0   
4  2022000005       1.0       1.0       NaN       1.0       2.0      1.0   

   COLGSEX1  NUMADULT  LANDSEX1  NUMMEN  NUMWOMEN  RESPSLCT  SAFETIME  \
0       NaN       2.0       NaN     1.0       1.0       2.0       NaN   
1       NaN       2.0       NaN     1.0       1.0       2.0       NaN   
2       NaN       1.0       2.0     NaN       NaN       NaN       NaN   
3       NaN       3.0       NaN     2.0       1.0       2.0       NaN   
4       NaN       2.0       NaN     1.0       1.0       2.0       NaN   

   CTELNUM1  CELLFON5  CADULT1  CELLSEX1  PVTRESD3  CCLGHOUS  CSTATE1  \
0       NaN       NaN      NaN       NaN       NaN       NaN      NaN   
1       NaN       NaN      NaN       NaN       NaN       NaN      NaN   
2       NaN       NaN      NaN       NaN       NaN       NaN      NaN   
3       NaN       NaN      NaN       NaN       NaN       NaN      NaN   
4       NaN       NaN      NaN       NaN       NaN       NaN      NaN   

   LANDLINE  HHADULT  SEXVAR  GENHLTH  PHYSHLTH  MENTHLTH  POORHLTH  PRIMINSR  \
0       NaN      NaN       2      2.0      88.0      88.0       NaN      99.0   
1       NaN      NaN       2      1.0      88.0      88.0       NaN       3.0   
2       NaN      NaN       2      2.0       2.0       3.0       2.0       1.0   
3       NaN      NaN       2      1.0      88.0      88.0       NaN      99.0   
4       NaN      NaN       2      4.0       2.0      88.0      88.0       7.0   

   PERSDOC3  MEDCOST1  CHECKUP1  EXERANY2  SLEPTIM1  LASTDEN4  RMVTETH4  \
0       1.0       2.0       1.0       2.0       8.0       NaN       NaN   
1       2.0       2.0       8.0       2.0       6.0       NaN       NaN   
2       1.0       2.0       1.0       1.0       5.0       NaN       NaN   
3       1.0       2.0       1.0       1.0       7.0       NaN       NaN   
4       2.0       2.0       1.0       1.0       9.0       NaN       NaN   

   CVDINFR4  CVDCRHD4  CVDSTRK3  ASTHMA3  ASTHNOW  CHCSCNC1  CHCOCNC1  \
0       2.0       2.0       2.0      2.0      NaN       2.0       2.0   
1       2.0       2.0       2.0      2.0      NaN       1.0       1.0   
2       2.0       2.0       2.0      2.0      NaN       1.0       2.0   
3       2.0       2.0       2.0      1.0      1.0       2.0       2.0   
4       2.0       2.0       2.0      2.0      NaN       2.0       2.0   

   CHCCOPD3  ADDEPEV3  CHCKDNY2  HAVARTH4  DIABETE4  DIABAGE4  MARITAL  EDUCA  \
0       2.0       2.0       2.0       2.0       1.0      80.0      1.0    6.0   
1       2.0       2.0       2.0       2.0       3.0       NaN      3.0    4.0   
2       2.0       2.0       2.0       2.0       3.0       NaN      1.0    6.0   
3       2.0       2.0       2.0       1.0       3.0       NaN      1.0    4.0   
4       2.0       2.0       2.0       2.0       3.0       NaN      1.0    5.0   

   RENTHOM1  NUMHHOL4  NUMPHON4  CPDEMO1C  VETERAN3  EMPLOY1  CHILDREN  \
0       1.0       1.0       1.0       2.0       2.0      7.0      88.0   
1       1.0       1.0       2.0       1.0       2.0      2.0      88.0   
2       1.0       2.0       NaN       1.0       2.0      7.0      88.0   
3       1.0       2.0       NaN       1.0       2.0      7.0      88.

AT THIS POINT WE HAVE 445132 RECORDS AND 326 FEATURES

# **1.1 RELEVANT FEATURE SELECTION**

Given  fetures, I tried to select  risk factors (columns/questions) in the BRFSS related to these risk factors.
 I consult the [BRFSS 2022 Codebook](https://www.cdc.gov/brfss/annual_data/2022/zip/codebook22_llcp-v2-508.zip) to see the questions and information about the questions for better insight of datasets. I try to match the variable names in the codebook to the variable names in the dataset which I converted into a CSV file that was originally a [ASCII file from BRFSS](https://www.cdc.gov/brfss/annual_data/2022/files/LLCP2022ASC.zip)  . I also reference some of the same features chosen for a research paper by [Zidian Xie et al for Building Risk Prediction Models for Type 2 Diabetes Using Machine Learning Techniques using the 2014 BRFSS.](https://www.cdc.gov/pcd/issues/2019/19_0109.htm)

**SELECTED FEATURES:-**


**Response Variable / Dependent Variable:**
*   (Ever told) you have diabetes (If "Yes" and respondent is female, ask "Was this only when you were pregnant?". If Respondent says pre-diabetes or borderline diabetes, use response code 4.) --> DIABETE4

**Independent Variables:**

**BMI**
*   Body Mass Index (BMI) --> _BMI5

**Smoking**
*   Have you smoked at least 100 cigarettes in your entire life? [Note: 5 packs = 100 cigarettes] --> SMOKE100

**Pre-diabetes**
*   When was the last time you had a blood test for high blood sugar or diabetes by a doctor, nurse, or other health professional? --> PDIABTS1
*   Has a doctor or other health professional ever told you that you had prediabetes or borderline diabetes? (If “Yes” and respondent is female, ask: “Was this only when you were pregnant?”) --> PREDIAB2

**Other Chronic Health Conditions**
*   (Ever told) (you had) a depressive disorder (including depression, major depression, dysthymia, or minor depression)? --> ADDEPEV3
*   (Ever told) you had a stroke. --> CVDSTRK3
*   Respondents that have ever reported having coronary heart disease (CHD) or myocardial infarction (MI) --> _MICHD

**Physical Activity**
*   Adults who reported doing physical activity or exercise during the past 30 days other than their regular job --> _TOTINDA

**Inadequate sleep**
*    On average, how many hours of sleep do you get in a 24-hour period? --> SLEPTIM1


**Alcohol Consumption**
*   Heavy drinkers (adult men having more than 14 drinks per week and adult women having more than 7 drinks per week) --> _RFDRHV8

**Health Care**
*   Do you have any kind of health care coverage, including health insurance, prepaid plans such as HMOs, or government plans such as Medicare, or Indian Health Service?  --> _HLTHPLN
*   Was there a time in the past 12 months when you needed to see a doctor but could not because of cost? --> MEDCOST1
*   About how long has it been since you last visited a doctor for a routine checkup? --> CHECKUP1

**Health General and Mental Health**
*   Would you say that in general your health is: --> GENHLTH
*   Now thinking about your mental health, which includes stress, depression, and problems with emotions, for how many days during the past 30 days was your mental health not good? --> MENTHLTH
*   Now thinking about your physical health, which includes physical illness and injury, for how many days during the past 30 days was your physical health not good? --> PHYSHLTH
*   Do you have serious difficulty walking or climbing stairs? --> DIFFWALK

**Demographics**
*   Indicate sex of respondent. --> SEXVAR
*   Fourteen-level age category --> _AGEG5YR
*   What is the highest grade or year of school you completed? --> EDUCA
*   Is your annual household income from all sources: (If respondent refuses at any income level, code "Refused.") --> INCOME3

In [137]:
df_selected = df[['DIABETE4',  '_BMI5','SMOKE100','ADDEPEV3','CVDSTRK3', '_MICHD','CHECKUP1' ,'_TOTINDA','SLEPTIM1', '_RFDRHV8','_HLTHPLN', 'MEDCOST1','GENHLTH', 'MENTHLTH', 'PHYSHLTH', 'DIFFWALK','SEXVAR', '_AGEG5YR', 'EDUCA', 'INCOME3' ]]

In [138]:
df_selected.shape


(445132, 20)

In [139]:
pd.set_option('display.max_columns', 500)
df_selected.head()

DIABETE4   _BMI5  SMOKE100  ADDEPEV3  CVDSTRK3  _MICHD  CHECKUP1  _TOTINDA  \
0       1.0     NaN       2.0       2.0       2.0     2.0       1.0         2   
1       3.0  2657.0       2.0       2.0       2.0     2.0       8.0         2   
2       3.0  2561.0       2.0       2.0       2.0     2.0       1.0         1   
3       3.0  2330.0       1.0       2.0       2.0     2.0       1.0         1   
4       3.0  2177.0       2.0       2.0       2.0     2.0       1.0         1   

   SLEPTIM1  _RFDRHV8  _HLTHPLN  MEDCOST1  GENHLTH  MENTHLTH  PHYSHLTH  \
0       8.0         1         9       2.0      2.0      88.0      88.0   
1       6.0         1         1       2.0      1.0      88.0      88.0   
2       5.0         1         1       2.0      2.0       3.0       2.0   
3       7.0         1         9       2.0      1.0      88.0      88.0   
4       9.0         1         1       2.0      4.0      88.0       2.0   

   DIFFWALK  SEXVAR  _AGEG5YR  EDUCA  INCOME3  
0       2.0       2        13    6.0     99.0  
1       2.0       2        13    4.0      5.0  
2       2.0       2         8    6.0     10.0  
3       2.0       2        14    4.0     77.0  
4       2.0       2         5    5.0      5.0

## **1.2 CLEANING DATA**

# 1.2.1  Drop missing values

In [140]:
# Dropping missing values
df_selected =df_selected.dropna()
df_selected.shape

(380525, 20)

# 1.2.2 modify and cleaning values for making it  suitable to ML algorithms

In [141]:
# DIABETE4
# going to make this ordinal. 0 is for no diabetes or only during pregnancy and for pre-diabetes or borderline diabetes too, 1 is for yes diabetes
# Remove all 7 (dont knows)
# Remove all 9 (refused)
df_selected['DIABETE4'] = df_selected['DIABETE4'].replace({2:0, 3:0, 1:1, 4:0})
df_selected = df_selected[df_selected.DIABETE4 != 7]
df_selected = df_selected[df_selected.DIABETE4 != 9]
df_selected.DIABETE4.unique()

array([0., 1.])

In [142]:
#1 _BMI5 (no changes, just note that these are BMI * 100. So for example a BMI of 4018 is really 40.18)
df_selected['_BMI5'] = df_selected['_BMI5'].div(100).round(0)
df_selected._BMI5.unique()

array([ 27.,  26.,  23.,  22.,  28.,  29.,  33.,  25.,  32.,  45.,  24.,
        37.,  31.,  39.,  34.,  19.,  21.,  35.,  38.,  20.,  46.,  13.,
        30.,  41.,  36.,  17.,  57.,  47.,  43.,  51.,  18.,  44.,  40.,
        50.,  52.,  56.,  42.,  58.,  16.,  49.,  54.,  15.,  92.,  60.,
        53.,  48.,  59.,  62.,  55.,  61.,  77.,  68.,  14.,  86.,  63.,
        66.,  79.,  64.,  73.,  75.,  69.,  71.,  74.,  12.,  85.,  95.,
        67.,  72.,  65.,  76.,  91.,  89.,  70.,  93.,  98.,  80.,  81.,
        82.,  96.,  78.,  83.,  87.,  90.,  88.,  84.,  94.,  97., 100.])

In [143]:
#2 SMOKE100
# Change 2 to 0 because it is No
# Remove all 7 (dont knows)
# Remove all 9 (refused)
df_selected['SMOKE100'] = df_selected['SMOKE100'].replace({2:0})
df_selected = df_selected[df_selected.SMOKE100 != 7]
df_selected = df_selected[df_selected.SMOKE100 != 9]
df_selected.SMOKE100.unique()

array([0., 1.])

In [144]:
#3 pre-diabetes section

#3.1 PDIABTS1
# changing 8 to 0 since it never had high sugar level
# Remove all 7 (dont knows)
# Remove all 9 (refused)
# df_selected['PDIABTS1'] = df_selected['PDIABTS1'].replace({8:0})
# df_selected = df_selected[df_selected.PDIABTS1 != 7]
# df_selected = df_selected[df_selected.PDIABTS1 != 9]
# df_selected.PDIABTS1.unique()

In [145]:
#3.2 PREDIAB2
# changing 3 to 0 since no , 2 to 0 since during pregnancy and leave 1 as it is
# Remove all 7 (dont knows)
# Remove all 9 (refused)
# df_selected['PREDIAB2'] = df_selected['PREDIAB2'].replace({3:0,2:0})
# df_selected = df_selected[df_selected.PREDIAB2 != 7]
# df_selected = df_selected[df_selected.PREDIAB2 != 9]
# df_selected.PREDIAB2.unique()

In [146]:
#4. Chronical health condition section

#4.1  ADDEPEV3

#change 2 to 0 ,means never told/had depressive order
# Remove all 7 (dont knows)
# Remove all 9 (refused)

df_selected['ADDEPEV3'] = df_selected['ADDEPEV3'].replace({2:0})
df_selected = df_selected[df_selected.ADDEPEV3 != 7]
df_selected = df_selected[df_selected.ADDEPEV3 != 9]
df_selected.ADDEPEV3.unique()


array([0., 1.])

In [147]:
#4.2  CVDSTRK3

#change 2 to 0 ,means never told/had stroke
# Remove all 7 (dont knows)
# Remove all 9 (refused)

df_selected['CVDSTRK3'] = df_selected['CVDSTRK3'].replace({2:0})
df_selected = df_selected[df_selected.CVDSTRK3 != 7]
df_selected = df_selected[df_selected.CVDSTRK3 != 9]
df_selected.CVDSTRK3.unique()


array([0., 1.])

In [148]:
#4.3  _MICHD

#change 2 to 0 ,never reported having coronary heart disease (CHD) or myocardial infarction (MI)

df_selected['_MICHD'] = df_selected['_MICHD'].replace({2:0})
df_selected = df_selected[df_selected._MICHD != 7]
df_selected = df_selected[df_selected._MICHD != 9]
df_selected._MICHD.unique()


array([0., 1.])

In [149]:
#5 _TOTINDA
# 1 for physical activity
# change 2 to 0 for no physical activity
# Remove all 9 (don't know/refused)
df_selected['_TOTINDA'] = df_selected['_TOTINDA'].replace({2:0})
df_selected = df_selected[df_selected._TOTINDA != 9]
df_selected._TOTINDA.unique()

array([0, 1])

In [150]:
#6  Inadequate sleep -->   SLEPTIM1

# change 1-24 values to 3 classes  0 for (less than 6 hr sleep), 1  for (6<x<9 hrs sleep ) and 2 for ( more than 9 hr sleep)
#  remove all 77/99(don't know/refused)
df_selected['SLEPTIM1'] = df_selected['SLEPTIM1'].replace({1:0,2:0,3:0,4:0,5:0,6:1,7:1,8:1,9:1,10:2,11:2,12:2,13:2,14:2,15:2,16:2,17:2,18:2,19:2,20:2,21:2,22:2,23:2,24:2})
df_selected = df_selected[df_selected.SLEPTIM1 != 77]
df_selected = df_selected[df_selected.SLEPTIM1 != 99]
df_selected.SLEPTIM1.unique()

array([1., 0., 2.])

In [151]:
#7 Alcohol Consumption --> _RFDRHV8
#change 1 to 0 for not heavy drinkers and 2 to 1 for heavy drinkers
#  remove all 9(don't know/refused/missing)
df_selected['_RFDRHV8'] = df_selected['_RFDRHV8'].replace({1:0,2:1})
df_selected = df_selected[df_selected._RFDRHV8 != 9]
df_selected._RFDRHV8.unique()

array([0, 1])

In [152]:
#8 Health Care

#8.1 _HLTHPLN
# change 2 to 0 for not having any form of health insurance
#  remove all 9(don't know/refused/missing)
df_selected['_HLTHPLN'] = df_selected['_HLTHPLN'].replace({2:0})
df_selected = df_selected[df_selected._HLTHPLN != 9]
df_selected._HLTHPLN.unique()


array([1, 0])

In [153]:
#8.2 MEDCOST1
#change 1  to 0 for could not afford medical cost and 2 to 1 for able to afford
#  remove all 7,9(don't know/not sure,refused)
df_selected['MEDCOST1'] = df_selected['MEDCOST1'].replace({1:0,2:1})
df_selected = df_selected[df_selected.MEDCOST1 != 9]
df_selected = df_selected[df_selected.MEDCOST1 != 7]
df_selected.MEDCOST1.unique()


array([1., 0.])

In [154]:
#8.3 CHECKUP1
#not changed
#  remove all 7,8,9(don't know/not sure,never,refused)
df_selected = df_selected[df_selected.CHECKUP1 != 7]
df_selected = df_selected[df_selected.CHECKUP1 != 8]
df_selected = df_selected[df_selected.CHECKUP1 != 9]
df_selected.CHECKUP1.unique()

array([1., 2., 3., 4.])

In [155]:
#9 Health General and Mental Health

#9.1  GENHLTH
#NO change  range of(excellent to poor ) --> (1 to 5)
#  remove all 7,9(don't know,refused)
df_selected = df_selected[df_selected.GENHLTH != 7]
df_selected = df_selected[df_selected.GENHLTH != 9]
df_selected.GENHLTH.unique()


array([2., 4., 5., 3., 1.])

In [156]:
#9.2 MENTHLTH
# already in days so keep that, scale will be 0-30
# change 88 to 0 because it means none (no bad mental health days)
# remove 77 and 99 for don't know not sure and refused

df_selected['MENTHLTH'] = df_selected['MENTHLTH'].replace({88:0})
df_selected = df_selected[df_selected.MENTHLTH != 99]
df_selected = df_selected[df_selected.MENTHLTH != 77]
df_selected.MENTHLTH.unique()

array([ 3.,  0.,  9.,  5., 15., 20., 14., 10., 18.,  1.,  2., 30.,  4.,
        6.,  7., 25.,  8., 27., 21., 12., 13., 28., 29., 26., 17., 11.,
       16., 23., 22., 24., 19.])

In [157]:
#9.3  PHYSLTH
# already in days so keep that, scale will be 0-30
# change 88 to 0 because it means none (no bad physical health days)
# remove 77 and 99 for don't know not sure and refused
df_selected['PHYSHLTH'] = df_selected['PHYSHLTH'].replace({88:0})
df_selected = df_selected[df_selected.PHYSHLTH != 77]
df_selected = df_selected[df_selected.PHYSHLTH != 99]
df_selected.PHYSHLTH.unique()

array([ 2.,  1.,  0.,  8.,  5., 30.,  4., 23., 14., 15.,  3., 10.,  7.,
       25.,  6., 21., 20., 29., 16.,  9., 27., 28., 13., 12., 11., 26.,
       17., 24., 18., 19., 22.])

In [158]:
#9.4 DIFFWALK

# change 2 to 0 for no. 1 is already yes
# remove 7 and 9 for don't know not sure and refused
df_selected['DIFFWALK'] = df_selected['DIFFWALK'].replace({2:0})
df_selected = df_selected[df_selected.DIFFWALK != 7]
df_selected = df_selected[df_selected.DIFFWALK != 9]
df_selected.DIFFWALK.unique()

array([0., 1.])

In [159]:
#10  DEMOGRAPHICS


#10.1  SEXVAR

# in other words - is respondent male (somewhat arbitrarily chose this change because men are at higher risk for diabetes disease)
# change 2 to 0 (female as 0). Male is 1
df_selected['SEXVAR'] = df_selected['SEXVAR'].replace({2:0})
df_selected.SEXVAR.unique()

array([0, 1])

In [160]:
#10.2  _AGEG5YR

# already ordinal. 1 is 18-24 all the way up to 13 wis 80 and older. 5 year increments.
# remove 14 because it is don't know or missing
df_selected = df_selected[df_selected._AGEG5YR != 14]
df_selected._AGEG5YR.unique()

array([ 8,  5, 13, 11, 10,  9, 12,  7,  6,  4,  3,  2,  1])

In [161]:
#10.3 EDUCA
# This is already an ordinal variable with 1 being never attended school or kindergarten only up to 6 being college 4 years or more
# Scale here is 1-6
# Remove 9 for refused:
df_selected = df_selected[df_selected.EDUCA != 9]
df_selected.EDUCA.unique()

array([6., 5., 4., 3., 2., 1.])

In [162]:
#10.4 INCOME3
# Variable is already ordinal with 1 being less than $10,000 all the way up to 11 being $200,000 or more
# Remove 77 and 99 for don't know and refused
df_selected = df_selected[df_selected.INCOME3 != 77]
df_selected = df_selected[df_selected.INCOME3 != 99]
df_selected.INCOME3.unique()

array([10.,  5.,  8.,  7.,  9.,  6.,  2.,  4.,  3.,  1., 11.])

# **1.3 changing feature names to understandable**

In [168]:
#Rename the columns to make them more readable
BRFSS22Cleaned = df_selected.rename(columns = {'DIABETE4':'DiabetesBinary',  '_BMI5':'BMI', 'SMOKE100':'Smoker',
                                            'ADDEPEV3':'Depression','CVDSTRK3':'Stroke', '_MICHD':'HeartDiseaseorAttack',
                                               'CHECKUP1':'RoutineCheckupSpan', '_TOTINDA':'PhysActivity',
                                               'SLEPTIM1':'SleepTime','_RFDRHV8':'HvyAlcoholConsump',
                                            '_HLTHPLN':'AnyHealthcare', 'MEDCOST1':'NoDocbcCost',
                                         'GENHLTH':'GenHlth', 'MENTHLTH':'MentHlth', 'PHYSHLTH':'PhysHlth', 'DIFFWALK':'DiffWalk',
                                         'SEXVAR':'Sex', '_AGEG5YR':'Age', 'EDUCA':'Education', 'INCOME3':'Income' })


In [169]:
BRFSS22Cleaned.head()

DiabetesBinary   BMI  Smoker  Depression  Stroke  HeartDiseaseorAttack  \
2             0.0  26.0     0.0         0.0     0.0                   0.0   
4             0.0  22.0     0.0         0.0     0.0                   0.0   
6             0.0  23.0     1.0         0.0     0.0                   0.0   
7             0.0  28.0     0.0         0.0     0.0                   0.0   
9             1.0  29.0     0.0         0.0     0.0                   0.0   

   RoutineCheckupSpan  PhysActivity  SleepTime  HvyAlcoholConsump  \
2                 1.0             1        0.0                  0   
4                 1.0             1        1.0                  0   
6                 1.0             1        1.0                  0   
7                 1.0             0        1.0                  0   
9                 1.0             1        1.0                  0   

   AnyHealthcare  NoDocbcCost  GenHlth  MentHlth  PhysHlth  DiffWalk  Sex  \
2              1          1.0      2.0       3.0       2.0       0.0    0   
4              1          1.0      4.0       0.0       2.0       0.0    0   
6              1          1.0      2.0       0.0       0.0       0.0    0   
7              1          1.0      3.0       0.0       0.0       0.0    0   
9              1          1.0      3.0       0.0       1.0       0.0    0   

   Age  Education  Income  
2    8        6.0    10.0  
4    5        5.0     5.0  
6   13        6.0     8.0  
7   13        4.0     7.0  
9   11        6.0     7.0

In [170]:
BRFSS22Cleaned.shape


(281116, 20)

In [171]:
BRFSS22Cleaned.groupby(['DiabetesBinary']).size()

DiabetesBinary
0.0    242692
1.0     38424
dtype: int64

# **2. SAVING CSV FILE**

In [173]:
BRFSS22Cleaned.to_csv('BRFSS22Cleaned.csv', sep=",", index=False)